### PR TITLE
doc: How to use IAM policies using APIs

### PR DESCRIPTION
**Pull request by sebferrer**
| Question         | Answer
| ---------------- | ---
| Branch?          |  doc/iam-policies-api
| Bug fix?         |  no
| New feature?     |  yes
| Breaking change? |  no
| Tickets          |  ECIAM-625
| License          |  BSD 3-Clause

## Description

This documentation is about how to use OVHcloud IAM policies using APIs.
It covers what are our policies and explains urn from identities, resources and actions. 

It explains how to use the api as well.

### CODE REVIEW

File: pages/account/customer/iam-policies-api/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+#### Create users
+
+List all the current users related to the account by calling **/me/identity/user**,
+
+**/me/identity/user**
+
+```json
+[
+  "user1",
+  "user2",
```
Comment (deathiop) [inline pages/account/customer/iam-policies-api/guide.en-gb.md]:
```suggestion
  "user2"
```

File: pages/account/customer/iam-policies-api/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+```json
+[
+    "account",
+    "packXdsl",
+    "cloudDB",
+    "loadbalancer",
+    "mxplanAccount",
+    "publicCloud",
+    "voipSmsvn",
+    "vps",
```
Comment (deathiop) [inline pages/account/customer/iam-policies-api/guide.en-gb.md:538]:
```suggestion
    "vps"
```

Comment (sebferrer) [reply pages/account/customer/iam-policies-api/guide.en-gb.md:538]: This one was on purpose to specify that there is actually way more output here. At first we put "..." but as it's a json error it was highlighted in red

File: pages/account/customer/iam-policies-api/guide.en-gb.md (hunk 1 above)
Comment (sebferrer) [reply pages/account/customer/iam-policies-api/guide.en-gb.md]: Done

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @sebferrer

**Pull request merged**